package.json

```diff
@@ -0,0 +1,131 @@
+{
+  "name": "@oneblink/apps-react",
+  "description": "Helper functions for OneBlink apps in ReactJS.",
+  "version": "0.1.0",
+  "author": "OneBlink <developers@oneblink.io> (https://oneblink.io)",
+  "bugs": {
+    "url": "https://github.com/oneblink/apps-react/issues"
+  },
+  "dependencies": {
+    "@babel/core": "^7.11.1",
+    "@babel/plugin-transform-flow-strip-types": "^7.10.4",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-flow": "^7.10.4",
+    "@babel/preset-react": "^7.10.4",
+    "@blinkmobile/camera": "^0.0.4",
+    "@blinkmobile/canvas-manipulation": "^1.0.1",
+    "@blinkmobile/geolocation": "^2.0.3",
+    "@oneblink/apps": "0.1.0-beta.13",
+    "@react-google-maps/api": "^1.9.12",
+    "abort-controller": "^3.0.0",
+    "abortcontroller-polyfill": "^1.5.0",
+    "babel-loader": "^8.1.0",
+    "blueimp-load-image": "^5.14.0",
+    "bulma": "^0.8.2",
+    "bulma-badge": "3.0.0",
+    "bulma-slider": "^2.0.4",
+    "bulma-steps": "^2.2.1",
+    "bulma-toast": "^2.0.1",
+    "bulma-tooltip": "^3.0.2",
+    "clsx": "^1.1.1",
+    "copy-to-clipboard": "^3.3.1",
+    "copy-webpack-plugin": "^6.0.3",
+    "cordova": "^10.0.0",
+    "cordova-android": "^9.0.0",
+    "cordova-ios": "^6.1.0",
+    "cordova-plugin-camera": "^4.1.0",
+    "cordova-plugin-device": "^2.0.3",
+    "cordova-plugin-exit": "^1.0.3",
+    "cordova-plugin-file": "^6.0.2",
+    "cordova-plugin-file-opener2": "^3.0.4",
+    "cordova-plugin-geolocation": "^4.0.2",
+    "cordova-plugin-ionic-webview": "^5.0.0",
+    "cordova-plugin-network-information": "^2.0.2",
+    "cordova-plugin-splashscreen": "^6.0.0",
+    "cordova-plugin-statusbar": "^2.4.3",
+    "cordova-plugin-whitelist": "^1.3.4",
+    "core-js": "^3.6.5",
+    "css-loader": "^4.2.1",
+    "dotenv": "^8.2.0",
+    "element-remove": "^1.0.4",
+    "escape-string-regexp": "^4.0.0",
+    "file-loader": "^6.0.0",
+    "file-saver": "^2.0.2",
+    "flatpickr": "4.6.3",
+    "getusermedia": "^2.0.1",
+    "html-loader": "^1.1.0",
+    "html-webpack-plugin": "^4.3.0",
+    "jsqr": "^1.3.1",
+    "jwt-decode": "^2.2.0",
+    "local-storage": "^2.0.0",
+    "lodash.clonedeep": "^4.5.0",
+    "lodash.debounce": "^4.0.8",
+    "lodash.throttle": "^4.1.1",
+    "morph-expressions": "^1.1.1",
+    "node-sass": "^4.14.1",
+    "offline-plugin": "^5.0.7",
+    "phonegap-plugin-barcodescanner": "^8.1.0",
```

Comment:
i think make the cordova stuff either peer or not here at all and document that it should be included if intending to create a native app. Im leaning towards to the latter, prob worth opening up to the wide team.